articles/virtual-machines/linux/login-using-aad.md

```diff
@@ -110,7 +110,7 @@ az vm extension set \
     --vm-name myVM
 ```
 
-The *provisioningState* of *Succeeded* is shown once the extension is successfully installed on the VM.
+The *provisioningState* of *Succeeded* is shown once the extension is successfully installed on the VM. The VM needs a running VM Agent to install the extension. Read more on [VM Agent Overview](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/agent-windows)
```

Comment:
Looks good, though need to remove the en-us locale from the outbound link per the following suggestions:

```suggestion
The *provisioningState* of *Succeeded* is shown once the extension is successfully installed on the VM. The VM needs a running VM Agent to install the extension. Read more on [VM Agent Overview](https://docs.microsoft.com/azure/virtual-machines/extensions/agent-windows)
```